### PR TITLE
fix/onboarding-no-more-flaky-tests

### PR DESCRIPTION
**Pull request by alannnc**
## What does this PR do?

Improvements on onboarding e2e test.
Also fixed change-passwrod e2e.

**Environment**: Staging(main branch) 

## Type of change

- [X] Chore (refactoring code, technical debt, workflow improvements)

### CODE REVIEW

**Comment by alannnc** [comment]
It's all green now, let's wait for a few updates on main so test run multiple times to find any flaky test due to server speed.

**Comment by PeerRich** [comment]
failing again. not sure if its onboarding or something different

File: apps/web/playwright/change-password.e2e.ts [hunk 1]
```diff
@@ -14,10 +14,16 @@ test.describe("Change Passsword Test", () => {
     if (!pro.username) throw Error("Test user doesn't have a username");
 
     // Fill form
-    await page.fill('[name="current_password"]', pro.username);
-    await page.fill('[name="new_password"]', `${pro.username}1111`);
-    await page.press('[name="new_password"]', "Enter");
+    // Wait for selector had to be used as there was a time we tried to fill but element was not ready
+    await page.waitForSelector("input[name='current_password']");
```
Comment (alannnc) [inline apps/web/playwright/change-password.e2e.ts:18]: @zomars This is the change to don't skip any fill input.

Comment (emrysal) [reply apps/web/playwright/change-password.e2e.ts:18]: Not the entire fix, I have reached the same conclusion in https://github.com/calcom/cal.com/pull/4247 but that only fixes part of the flakyness.

**Review by zomars — APPROVED** [review]

**Pull request merged**